src/modules/transactions.ts

```diff
@@ -730,7 +730,7 @@ Transactions.prototype.undoUnconfirmedList = function (cb) {
  * @param {function} cb - Callback function
  */
 Transactions.prototype.apply = function (transaction, block, sender, cb) {
-    library.logger.debug('Applying confirmed transaction', transaction.id);
+    library.logger.debug(`[Transactions][apply] transaction id: ${transaction.id}}`);
```

Comment:
Perhaps transactionId it's not enough for information about problem and you have to show all transaction's info?
It's for all similar places!
```suggestion
    library.logger.debug(`[Transactions][apply] transaction: ${JSON.stringify(transaction}}`);
```